### PR TITLE
Reorganization project

### PR DESCRIPTION
**Pull request by mrmlnc**
Suggest reorganizing the project.

Common structure:

 * `dist` - distribution (dist.js + dist.min.js)
 * `test` - testing your project 

This structure uses the most packages.

### CODE REVIEW

File: dist/notification.js [hunk 1]
```diff
@@ -1 +1,94 @@
...
+
+		if (commonJS) {
+			module.exports = Notification;
+		} else {
+			window.Notification = Notification;
+		}
+		
+	}
+
+})(window);
```
Comment (drKraken) [inline dist/notification.js]: Seems you forgot new line.

File: gulpfile.js [hunk 2]
```diff
@@ -15,16 +16,19 @@ gulp.task('js', function () {
...
 });
 
-//init browser sync via live reloads
+// init browser sync via live reloads
 gulp.task('browser', function () {
     browserSync.init(['*.html', '*.js'], {
         server: {
-            baseDir: './'
+            baseDir: ['./test/', './dist/']
         }
```
Comment (drKraken) [inline gulpfile.js]: Why this?

File: dist/notification.js (hunk 1 above)
Comment (mrmlnc) [inline dist/notification.js]: It's copied your file: [line 94](https://github.com/drKraken/notification-fallback.js/blob/master/notification.js#L94)

File: gulpfile.js (hunk 2 above)
Comment (mrmlnc) [inline gulpfile.js]: The test file is located in the directory `test/`.
The script file is located in the directory `dist/`.

The plugin (Browser Sync) gives you access to only the specified directory and its subdirectories. We have different folders. So need to specify both folders, either root directory.

**Pull request merged**